aws-amplifyuibuilder-component/pom.xml

```diff
@@ -24,6 +24,7 @@
         <dependency>
             <groupId>software.amazon.cloudformation</groupId>
             <artifactId>aws-cloudformation-rpdk-java-plugin</artifactId>
+            <version>[2.0.0,3.0.0)</version>
```

Comment:
mismatched braces

Reply:
That's on purpose. 

```
A square bracket ( [ & ] ) means "closed" (inclusive).
A parenthesis ( ( & ) ) means "open" (exclusive).
```

Nuget is the same